### ISSUE TITLE
Brew'd app not starting

### ISSUE THREAD

**Issue by kevinpapst** [AlexandrePTJ/kemai#103]
Just tried the tap and the cask installation was easy (if you know how to get around th security warnings).
But the first start resulted in a "bouncing app icon" for about a minute.

I forced shutdown of the app and restarted: again bouncing and bouncing and bouncing (in the app tray).
Finally it stopped bouncing, but there is no app window (nor any "quick-start" dropdown - not sure how the apps/icons ion the upper right toolbar are called).

How to debug what is wrong?

**Comment by AlexandrePTJ** [comment]
Log files are stored in /Users/<User>/Library/Application Support/Kemai/Kemai.
Please post content.

**Comment by kevinpapst** [comment]
Not existing:
<img width="816" alt="Bildschirmfoto 2023-07-23 um 20 47 16" src="https://github.com/AlexandrePTJ/kemai/assets/533162/0f22ab80-aa28-44fe-9c7d-fb1b854f1c32">
🤷 

I think I has a manually installed app before. Could it be, that there is some kind of "leftover" from the old one that is now blocking?

![out](https://github.com/AlexandrePTJ/kemai/assets/533162/6757a3e6-294c-4d32-b4f4-2d47db4280d9)

**Comment by AlexandrePTJ** [comment]
Yes it could be a leftover from a previous version. Config file should be located at : /Users/kevin/Library/Preferences/Kemai.
First versions use ini file, now it is a json. I may have missed something during the migration process.

**Comment by kevinpapst** [comment]
Nope, this directory is also not existing:

<img width="531" alt="Bildschirmfoto 2023-07-23 um 22 25 18" src="https://github.com/AlexandrePTJ/kemai/assets/533162/701afa4e-10a0-4504-9dec-96dba4c5e930">

Any other idea?

```bash
brew remove kemai
brew install kemai
```

did not change anything...

**Comment by kruegge82** [comment]
for me is also not working in MacOs Ventura 13.5.2
my app icon is not bouncing but i get also no error or window
in ApplicationSupport also no Kemai folder

**Comment by maxguru** [comment]
I was having a similar issue on Gentoo Linux after upgrading to the latest version (hang/freeze on startup with no sign of life), however, it seems to have resolved itself after I fiddled around with older versions. Not sure what was causing it, maybe old version data was causing an infinite loop in the new version of the app until I ran older versions that somehow corrected it.

**Comment by AlexandrePTJ** [comment]
I will be able to work on dmg packaging and fix config migrations issues.
Also please check https://github.com/AlexandrePTJ/kemai/discussions/134
